src/ios/SocketsForCordova/Classes/SocketAdapter.m

```diff
@@ -34,17 +34,16 @@
 
 int const WRITE_BUFFER_SIZE = 10 * 1024;
 
-int openTimeoutSeconds = 5.0;

```

Comment:
Wrong timeout is changed here, openTimeoutSeconds = timeout for opening, we need to change the write timeout